cobalt/shell/browser/shell_browser_context.cc

```diff
@@ -83,24 +81,6 @@ void ShellBrowserContext::InitWhileIOAllowed() {
   if (cmd_line->HasSwitch(switches::kIgnoreCertificateErrors)) {
     ignore_certificate_errors_ = true;
   }
-  if (cmd_line->HasSwitch(switches::kContentShellDataPath)) {
```

Comment:
We should follow https://chromium-review.googlesource.com/c/chromium/src/+/6480975 and add this code to shell_paths.cc